packages/nimble-components/src/anchor/tests/anchor.stories.ts

```diff
@@ -56,12 +58,19 @@ const metadata: Meta<AnchorArgs> = {
             control: { type: 'radio' },
             description:
                 'Set to `prominent` to make the anchor appear in a different color than normal text.'
+        },
+        contenteditable: {
+            options: ['false', 'true'],
+            control: { type: 'radio' },
+            description:
+                'Set this to the string "true" (or set the attribute without any value)  when the anchor is within an editable region (i.e. element/hierarchy with [contenteditable](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/contenteditable)). Whereas native elements inherit their `contenteditable` value by default, the `nimble-anchor` requires this attribute be explicitly set. When set, the anchor cannot be focused or operated.'
```

Comment:
So when I set this to true in storybook I expect to be able to click inside and edit the text but nothing seems to happen on chrome on windows: https://60e89457a987cf003efc0a5b-csyyuqlmje.chromatic.com/?path=/story/components-anchor--anchor&args=label:link;underlineHidden:!false;contenteditable:true;appearance:default

Reply:
Looks like contenteditable needs to be set on the span around the link. Then I can use arrow keys to navigate onto the link and edit its contents.